backend/question-service/package.json

```diff
@@ -1,8 +1,10 @@
 {
   "name": "question-service",
   "version": "1.0.0",
-  "main": "index.ts",
+  "main": "app.ts",
```

Comment:
is the main file suppose to be `server.ts`?

Reply:
oh okay I can change to the entrypoint file instead